checkov/common/checks_infra/solvers/attribute_solvers/base_attribute_solver.py

```diff
@@ -88,3 +89,13 @@ def get_attribute_patterns(attribute: str) -> Tuple[Pattern[str], Pattern[str]]:
         pattern_without_index = re.compile(pattern)
 
         return pattern_with_index, pattern_without_index
+
+    @staticmethod
+    def _is_variable_dependant(value: Any, source: str) -> bool:
+        if source == 'Terraform' and is_terraform_variable_dependent(value):
+            return True
+        # TODO add logic for CloudFormation
+        # elif source == 'CloudFormation' and is_cloudformation_variable_dependent(value):
+        #     return True
```

Comment:
Because the rendering logic for CFN is quite different and requires its own analysis which will take some more time, and the customer is impatiently waiting for the TF behavior change.